_posts/en/newsletters/2020-02-19-newsletter.md

```diff
@@ -0,0 +1,298 @@
+---
+title: 'Bitcoin Optech Newsletter #85'
+permalink: /en/newsletters/2020/02/19/
+name: 2020-02-19-newsletter
+slug: 2020-02-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests help testing release candidates for
+Bitcoin Core and C-Lightning, summarizes a discussion about taproot
+versus implementing MAST and schnorr signatures separately, describes new ideas for
+using PoDLEs in LN channel construction, and highlights a new
+implication of work on privacy-enhanced payments to unannounced LN
+channels.  Also included are our regular sections about notable changes to
+popular services, client software, and infrastructure projects
+
+## Action items
+
+- **Help test Bitcoin Core 0.19.1rc2:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+- **Upgrade to C-Lightning 0.8.1:** this [release][cl 0.8.1]  adds
+  several new features (including those described in the *notable
+  changes* section below) and provides multiple bug fixes.  See the
+  [changelog][cl changelog] for detailed list of updates.
+
+FIXME:check for latest RCs / releases shortly before publication
+
+## News
+
+- **Discussion about taproot versus alternatives:** a group of
+  developers who prefers to remain anonymous (so we'll call them Anon)
+  wrote a [criticism][anon reflowed] of taproot in comparison to
+  alternative approaches for enabling [MAST][topic mast] and [schnorr
+  signatures][topic schnorr signatures] in Bitcoin.  Anon concludes
+  their criticism with five questions which we use below to organize our
+  summary of Anon's concerns and the replies posted by several Bitcoin
+  contributors.
+
+    1. {:#tap1} Anon asks, "Is taproot actually more private than bare
+       MAST and schnorr separately?  What are the actual anonymity set
+       benefits compared to doing them separately?"
+
+       Anthony Towns [replies][towns tap], "Yes [it is more private],
+       presuming single-pubkey-single-signature remains a common
+       authorization pattern."  Towns shows that single-sig spends
+       currently represent more than 57% of all transaction outputs (and
+       possibly much more, given the frequent use of P2SH-wrapped
+       P2WPKH).  The number of users satisfied with single-sig will
```

Comment:
"users satisfied with" sounds off to me. Perhaps "transactions satisfied with" or "transactions using" or other?